extensions/aissemble-extensions-model-training-api-sagemaker/pyproject.toml

```diff
@@ -12,7 +12,7 @@ packages = [
 python = ">=3.11.4, <3.12"
 fastapi = ">=0.95.0"
 uvicorn = {version = "^0.18.0", extras = ["standard"]}
-pydantic = ">=1.8.0,<2.0.0"
+pydantic = "^2.8.0"
 kubernetes = ">=26.1.0"
```

Comment:
In general, if we use `>=` we should see some better performance. When we use `^`, it limits the number of potential matches greatly. This is a good discussion Ryan posted a while back on the upper limit notation: https://iscinumpy.dev/post/bound-version-constraints/#tldr